src/components/Intro.js

```diff
@@ -93,10 +90,11 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     direction: 'row',
     alignItems: 'center',
-    justifyContent: 'flex-start',
+    justifyContent: 'center',
     padding: theme.spacing(1),
```

Comment:
The numbered circles should still align at the top even if the steps differ in the length of text ([mock here](https://www.figma.com/file/KCnKxjXCMFjYx1Mi6wZe3B/Tab-for-Seas-Web-Pages?node-id=12%3A442)):
https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-61b7dbed4af8a9ac39fc39ff-1200&appId=6154d1bab7cd94003aee223d&number=249&view=changes

Reply:
easiest way to do this is use the 'align-self' property on the circle itself

Reply:
also don't forget to adjust the circle text to typography ie <Avatar className={cx.numberCircle}>
              <Typography>1</Typography>
            </Avatar>